apps/web/src/components/HeaderBanner/index.tsx

```diff
@@ -0,0 +1,137 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+const HeaderBanner = () => {
+  return (
+    <div
+      className="hero-bbo-header flex flex-col relative items-center justify-center w-full h-[16.25rem]
+                 rounded-[12px] border-[var(--colors-common-ash)] border-[2px] bg-[var(--primary-charcoal)]
+                 py-[2rem] px-[2rem] mt-[4rem]"
+    >
+      <div
+        className="header-bbo-flip-container group perspective-[10000px] flex items-center justify-center
+                   w-[78rem] h-[7.4375rem]"
+      >
+        <div
+          className="relative w-full h-full transition-transform duration-600 transform-3d
+                     group-hover:rotate-y-180"
+        >
+          {/* Front side */}
+          <div className="absolute inset-0 backface-hidden flex items-center justify-center">
+            <div
+              className="text-headline bg-gradient-to-r from-[var(--secondary-blue)] to-[var(--secondary-purple)]
+                         bg-clip-text text-transparent"
+            >
+              <h1 className="main-heading" aria-label="Main heading: Billionaire Blast Off">
+                #BillionaireBlastOff
+              </h1>
+            </div>
+          </div>
+          {/*Back side */}
+          <div className="absolute inset-0 max-w-full backface-hidden flex items-center justify-center rotate-y-180">
+            <div
+              className="text-headline-back bg-gradient-to-r from-[var(--secondary-yellow)] to-[var(--primary-orange)]
+                         bg-clip-text text-transparent"
+            >
+              <h2 className="flip-heading" aria-label="Alternative heading: Open What You Want">
+                #OpenWhatYouWant
+              </h2>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <span className="powered-by relative flex w-full justify-end items-center">
+        <p className="mr-[8px] font-[family-name:var(--sharp-sans)] text-[10px] font-medium tracking-[0.8] z-20">
```

Comment:
convert all `px` to `rem`

Also it might make sense to have the word "Firefox" in the paragraph text but invisible (like font-size: 0.001, opacity: 0) for SEO purposes...